algorithms/kernel/kernel_function/kernel_function_linear_dense_default_impl.i

```diff
@@ -120,8 +120,8 @@ services::Status KernelImplLinear<defaultDense, algorithmFPType, cpu>::computeIn
 {
```

Comment:
Overall comment - you optimized linear kernel only. But DAAL has RBF kernel as well.
Do we see the regression in SVM with RBF kernel?

Reply:
Yes, I see a place in RBF kernel where the regression may be. So I'm going to optimize it too

Reply:
Okay, let's do it